ui/src/router/edit/EditRouterView.js

```diff
@@ -8,12 +8,16 @@ import { VersionComparisonView } from "./components/VersionComparisonView";
 import { useTuringApi } from "../../hooks/useTuringApi";
 import { DeploymentSummary } from "../components/form/components/DeploymentSummary";
 import { ConfirmationModal } from "../../components/confirmation_modal/ConfirmationModal";
+import { VersionCreationSummary } from "../components/form/components/VersionCreationSummary";
 
 const EditRouterView = ({ projectId, currentRouter, ...props }) => {
   const { data: updatedRouter } = useContext(FormContext);
   const [showDiffView, toggleDiffView] = useToggle();
 
-  const [submissionResponse, submitForm] = useTuringApi(
+  const [
+    submissionCreateVersionWithDeploymentResponse,
```

Comment:
nit: For brevity, I'd prefer to distinguish them as `updateRouterResponse` vs `createRouterVersionResponse`. Implicitly, it's understood that update router will update the deployment as well and create version only creates a version.

Reply:
Alright, that makes sense - internally it aligns with the `create` and `update` terms that we'll be using in the backend too.